build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -153,10 +153,15 @@ public void CrossgenDirectory(BuildTargetContext c, string pathToAssemblies)
 
                 IList<string> crossgenArgs = new List<string> {
                     "-readytorun", "-in", file, "-out", tempPathName,
-                    "-JITPath", GetLibCLRJitPathForVersion(),
                     "-platform_assemblies_paths", platformAssembliesPaths
                 };
 
+                if (CurrentPlatform.IsUnix)
+                {
+                    crossgenArgs.Add("-JITPath");
```

Comment:
really?  Is this what @schellap said to do?  Why do we only require -JITPath on non-Windows?

/cc @gkhanna79